docs/dockstore-introduction.rst

```diff
@@ -1,138 +1,46 @@
-
-Dockstore also attempts to work with new and alternative
-languages/standards such as `Nextflow <https://www.nextflow.io/>`__ as
-popular challengers to CWL and WDL. We also work on the `GA4GH Tool
-Registry <https://github.com/ga4gh/tool-registry-service-schemas>`__ standard as
-a way of sharing data with workflow platforms and partners. We are also
-working with the task execution, workflow execution, and data transfer
-standards developing at the GA4GH.
-
-Built with Docker and Git
--------------------------
-
-.. note:: See `Docker Overview <https://docs.docker.com/get-started/overview/>`__ for an excellent overview about Docker.
-
-Docker repositories, like `Docker Hub <https://hub.docker.com/>`__,
-`Quay.io <https://quay.io/>`__ and `GitLab <https://about.gitlab.com>`__, and
-source control repositories like `GitHub <https://github.com>`__,
-`Bitbucket <https://bitbucket.org/>`__, and
-`GitLab <https://about.gitlab.com>`__, provide much of the infrastructure we
-need. Docker repositories allow users to build, publish, and share both
-public and private Docker images. However, the services lack
-standardized ways of describing how to invoke tools contained within
-Docker containers. Workflow descriptor languages provide standardised
-ways to define the inputs, parameterizations, and outputs of tools in a
-controlled way. Together, these resources provide the necessary tools to
-share analytical tools in a highly portable way, a key concern for the
-scientific community.
-
-.. figure:: /assets/images/docs/Ways_to_get_into_Dockstore.png
-   :alt: Overview
-
-   Overview
-
-Strategies
-----------
-
-You can register your tools and workflows on Dockstore in three broad
-ways as depicted above.
-
-A) Following our
-   :doc:`tutorials <getting-started/getting-started>`,
-   you can save your descriptors on GitHub, build your Docker image
-   automatically on Quay.io, and have Dockstore reach out and index your
-   tools
-
-B) Dockstore can retrieve your workflow descriptors from GitHub and
-   other source control methods. You are responsible for ensuring that
-   your descriptors point at valid Docker images.
-
-C) You will be able to use our new hosted workflows service to store
-   tools and workflows directly on dockstore.org to quickly get started,
-   prototype your ideas, and share workflows with a limited audience.
-
-In all three cases, you will have an opportunity to clean-up and
-configure your work before publishing to the rest of the world to see.
-
-You can mix and match in a number of these approaches. For example, you
-can go beyond our simple tutorials and automated approach to manually
-register tools that point at locations like Docker Hub, Seven Bridges,
-and Amazon ECR. You can substitute WDL and Nextflow for the descriptor
-language for workflows. You might even be able to mix and match
-descriptor languages eventually!
-
-Over time, we find "skinny" Docker images, those with single tools
```

Comment:
Move this discussion on "fat" and "skinny" to the FAQ or something, I feel like I may need to reference it some time in the near future but agree it no longer needs to be in intro

Reply:
I'm going to move this into the revamp of the Docker tutorial in a later PR.